legal-api/src/legal_api/resources/v2/business/colin_sync.py

```diff
@@ -56,6 +56,7 @@ def get_completed_filings_for_colin():
         if filing_json and filing.filing_type != 'lear_epoch' and \
                 (filing.filing_type != 'correction' or business.legal_type != Business.LegalTypes.COOP.value):
             filing_json['filingId'] = filing.id
+            filing_json['filing']['header']['date'] = filing.filing_date.isoformat()
```

Comment:
Which filings should be tested due to this change? Do you have screenshots?

Reply:
Generic to all filings

Reply:
Please let QA know in ticket test notes.